Makefile

```diff
@@ -13,10 +14,18 @@ GIT_HOOKS = $(wildcard $(GIT_HOOKS_PATH)/*)
 INSTALLED_TOOLCHAINS = $(shell $(RUSTUP) toolchain list)
 INSTALLED_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain $(TOOLCHAIN))
 INSTALLED_NIGHTLY_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain $(NIGHTLY_TOOLCHAIN))
-AVAILABLE_CARGO_COMMANDS = $(shell $(CARGO) --list)
```

Comment:
rust was only installed because of this command? 😅

Reply:
Yes :) `$(CARGO)` includes the installation of a toolchain.